tenable/io/v3/was/attachments/api.py

```diff
@@ -0,0 +1,69 @@
+'''
+Attachments
+===========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`attachments <was-v2-attachments>` API.
+
+Methods available on ``tio.v3.was.attachments``:
+
+.. rst-class:: hide-signature
+.. autoclass:: Attachment
+    :members:
+'''
+from io import BytesIO
+from typing import BinaryIO, Optional
+from uuid import UUID
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class AttachmentsAPI(ExploreBaseEndpoint):
+
+    _path = 'api/v3/was/attachments'
+
+    def download_attachment(
+        self,
+        attachment_id: UUID,
+        fobj: Optional[BinaryIO] = None
+    ) -> BinaryIO:
```

Comment:
Hi nishant, could u pls update the parameters as per the latest standard => please pick the first approach = https://github.com/dhvanilshah-tenable/pyTenable/pull/12#discussion_r769828173

Reply:
Done